scripts/rebuild.sh

```diff
@@ -3,19 +3,32 @@
 set -e
 
 cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
-if [ ! -e ../vcpkg ]
+repo_url=https://github.com/Microsoft/vcpkg
+vcpkg_commit=$(git ls-remote $repo_url | head -n1 | awk '{ print $1 }')
+filename="commit.txt"
+if [ ! -f "$filename" ]
+then
+    # create file with default value
+    echo "0" > "$filename"
+    echo "File created with default value: $filename"
+fi
+
+content=$(cat commit.txt)
+
+if [ $vcpkg_commit != $content ]
```

Comment:
This algorithm is still 100% dynamic.

In order for this to not be dynamic, this file should _never_ have any capability to use latest. The file commit.txt should be updated in a different pipeline step.
```suggestion
vcpkg_commit=$(cat commit.txt)

if [ ! -e ../vcpkg ]
```

Reply:
Can you fix this?